scc-big-data.ipynb

```diff
@@ -0,0 +1,1883 @@
+    "nbgrader": {
+     "cell_type": "markdown",
+     "checksum": "03942a1f77b120d3fabb5ba62a1e0a4d",
+     "grade": false,
+     "grade_id": "step-2",
+     "locked": true,
+     "schema_version": 3,
+     "solution": false,
+     "task": false
+    },
+    "tags": []
+   },
+   "source": [
+    "## STEP 2 - Download/Access Urban Greenspaces and Census Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab571d63-d20c-456e-885f-e7d3ede0deb7",
+   "metadata": {
+    "deletable": false,
+    "editable": false,
+    "nbgrader": {
+     "cell_type": "markdown",
+     "checksum": "474974107ac5e9b9594451476edf7bb2",
+     "grade": false,
+     "grade_id": "chicago-task",
+     "locked": true,
+     "points": 3,
+     "schema_version": 3,
+     "solution": false,
+     "task": true
+    },
+    "tags": []
+   },
+   "source": [
+    "### Download City of Chicago Boundary\n",
+    "\n",
+    "You can find the City of Chicago Boundary on the [City of Chicago Data Portal](https://data.cityofchicago.org/). \n",
+    "\n",
+    "> **Make sure to download in `Original` format**, as the `Shapefile` format has not been working lately.\n",
+    "\n",
+    "**YOUR TASK:**\n",
+    "\n",
+    "  1. Download the City of Chicago Boundary\n",
+    "  2. Use a **conditional statement** to cache the boundary at a **reproducible file path**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93ba244e-6621-4c06-86f9-2023b5cd4474",
+   "metadata": {
+    "deletable": false,
+    "nbgrader": {
+     "cell_type": "code",
+     "checksum": "f351e3dccbfe7948da3c162d55613850",
+     "grade": false,
+     "grade_id": "chicago-ans",
+     "locked": false,
+     "schema_version": 3,
+     "solution": true,
+     "task": false
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Download the City of Chicago boundary and put it in folder. \n",
+    "# If the boundary has already been downloaded then use the file\n",
+    "#   from the folder.\n",
+    "\n",
+    "chi_path = os.path.join(proj_dir, 'chicago_boundary')\n",
+    "if not os.path.exists(chi_path):\n",
+    "    chi_url = (\"https://data.cityofchicago.org/api\"\n",
+    "               \"/geospatial/ewy2-6yfk?method=export&format=Original\") \n",
+    "    gpd.read_file(chi_url).to_file(chi_path)\n",
+    "    \n",
+    "chi_boundary = gpd.read_file(chi_path)\n",
+    "chi_boundary.plot()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "73640b34-5711-4125-bcc1-948b73a9d874",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# What projection is the Chicago shapefile set to?\n",
+    "chi_boundary.crs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ec26780-8f99-406c-9899-09fe85f15a67",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Reproject to WGS 84 -- WGS84 - World Geodetic System 1984 to match NAIP imagery\n",
```

Comment:
It's great that you reprojected to match the NAIP data as the common crs for all the layers. I had some trouble matching the layers later on.